DataFormats/Math/test/CholeskyInvert_t.cu

```diff
@@ -162,8 +163,9 @@ void go(bool soa) {
 
 #ifndef DOPROF
       cudautils::launch(invertSeq<MX, DIM>, {blocksPerGrid, threadsPerBlock}, (MX *)(m_d.get()), SIZE);
+      cuda::launch(invertSeq<MX, DIM>, {blocksPerGrid, threadsPerBlock}, (MX *)(m_d.get()), SIZE);
```

Comment:
Looks like this line is there by mistake - possibly a leftover from the conflict resolution ?